### PR TITLE
add a list of programming languages

### PR DESCRIPTION
**Pull request by muffinista**
I generated a list of programming languages for my NaNoGenMo entry, and it was suggested that it might be a good thing to add to corpora, so here it is! I built this from the TIOBE top languages list and a few other similar sources.

### CODE REVIEW

**Comment by coleww** [comment]
https://github.com/dariusk/corpora/blob/master/data/technology/computer_sciences.json i think i put a list of programming languages under the name "computer sciences" because i am bad at naming things :laughing:

**Comment by DazzlingDevelopment** [comment]
@coleww 
Yeah, I don't know. Perhaps "computer sciences" should be split into different categories? For instance,
1. programmingLanguages.json (Contains different programming languages, such as @muffinista's list)
2. fields.json (Contains different fields in computer science)
3. etc.. (As more topics on computer science are added, we can create new files for them.)

**Comment by muffinista** [comment]
@coleww I'd skimmed your list and thought it was more about technologies in general instead of specific languages. I'm open to chucking my list or splitting it all up or anything else though.

**Comment by coleww** [comment]
@muffinista ah yeah upon reviewing that list it is mostly amazon web services and startups :laughing: :computer: 

 I can make a PR to move any languages from that into this list, and rename/re-describe the computer_sciences list to something clearer.

File: data/technology/programming_languages.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  "ABAP",
+  "ActionScript",
+  "Ada",
+  "Assembly language",
+  "Awk",
+  "C",
+  "C#",
+  "C++",
+  "COBOL",
+  "CSS",
```
Comment (coleww) [inline data/technology/programming_languages.json]: :metal:

**Pull request merged**